public/app/features/explore/TraceView/components/TracePageHeader/SpanFilters/SpanFilters.tsx

```diff
@@ -141,7 +141,7 @@ export const SpanFilters = memo((props: SpanFilterProps) => {
       <Collapse label={collapseLabel} collapsible={true} isOpen={showSpanFilters} onToggle={setShowSpanFilters}>
         <InlineFieldRow className={styles.flexContainer}>
           <InlineField label={t('explore.span-filters.label-service-name', 'Service name')} labelWidth={16}>
-            <HorizontalGroup spacing={'xs'}>
+            <Stack gap={3}>
```

Comment:
Based on [this](https://github.com/grafana/grafana/blob/f6e472f8797cde4dfdc3ab4f1fc3fc5969b7c8fe/packages/grafana-ui/src/components/Layout/Layout.tsx#L172) this should be 0.5
```suggestion
            <Stack gap={0.5}>
```